tests/integration-tests/tests/performance_tests/test_starccm/test_starccm/pcluster.config.yaml

```diff
@@ -0,0 +1,47 @@
+Region: eu-west-1
+Image:
+  Os: alinux2
+Imds:
+  ImdsSupport: v2.0
+HeadNode:
+  InstanceType: c5n.18xlarge
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    AdditionalIamPolicies:
+      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore # Required to report patching status
+Scheduling:
+  Scheduler: slurm
+  SlurmSettings:
+    ScaledownIdletime: 5
```

Comment:
In integ tests these propertie are not needed because we do not need to scale-up and down within the integ test.